### ISSUE TITLE
reload plugin list in add node

### ISSUE THREAD

**Issue by cancan101** [jtriley/StarCluster#489]
I changed my list of cluster plugins in the config file but running add node still uses the list from when cluster was started. Same applies to changing Ami.

Would be good to have flag to reload setting in add node.

**Comment by cancan101** [comment]
Partially addressed in #499

**Comment by vohtaski** [comment]
I guess we have a similar issue.
Had the following settings for plugin in config:

```
master_slots = 2
node_slots = 2
```

Now changed them to

```
master_slots = 0
node_slots = 2
```

When we do `starcluster restart` or `starcluster addnode`, it takes old data.
However, when we run `starcluster runplugin my_plugin`, it takes correct data.

Looking at the code,
first script is using this code and does not take numbers from config
https://github.com/jtriley/StarCluster/blob/develop/starcluster/cluster.py#L1690
second script take numbers from config here:
https://github.com/jtriley/StarCluster/blob/develop/starcluster/cluster.py#L379

question, where does it store the previous config that was used for starting the cluster?

**Comment by cancan101** [comment]
I believe these settings are stored in the cluster user data. This is where some of that is loaded: https://github.com/jtriley/StarCluster/blob/ca6933854ee4742d7fefe422c934516372bbd5e9/starcluster/cluster.py#L896

**Comment by vohtaski** [comment]
yep, thanks, I found it after. In the end we decided to kill the cluster and start again with new settings, maybe it's even better :) For now I have a quick fix to read config file from the plugin.

but I'm still `+1`, think it is useful for plugins
